unblob/processing.py

```diff
@@ -127,7 +123,11 @@ def _process_task(self, result: TaskResult, task: Task):
             log.debug("Ignoring empty file")
             return
 
-        _FileTask(self._config, task, size, result).process()
+        filetask = _FileTask(self._config, task, size, result)
+        filetask.process()
+        # ensure that the root extraction directory is created even for empty extractions
+        if task.depth == 0:
+            filetask.extract_dir.mkdir(parents=True, exist_ok=True)
```

Comment:
if we can not extract anything from the file, then do we really need that empty directory?

Reply:
I have checked it again, and previously in a few cases there were no output, and no extract directory was created. I still feel, that it is confusing and it is better to create the directory.